src/content/docs/tutorials/javareporter/java-check-tests.md

```diff
@@ -0,0 +1,147 @@
+        - Sync command usage
+        - Add IDs to test methods
+
+    ## CLI Commands and Options
+        - sync command (import + pull-ids)
+        - import command
+        - pull-ids command
+        - clean-ids command
+        - directory options
+
+    ## Configuration and Setup
+        - Environment variables (TESTOMATIO_URL, TESTOMATIO)
+        - One-liner sync commands for different OS
+        - Jar file management
+-->
+
+# Import tests with Java-Check-Tests CLI
+
+Java-check-tests is a CLI tool developed for convenient import of your test codebase (JUnit5 and TestNG) to Testomat.io.  
+The repository can be found [here](https://github.com/testomatio/java-check-tests/tree/main).  
+***It is important to sync your tests with the Testomat.io server to create proper test folder structure and  
+to be able to see the source code of your tests in the UI***
+
+## The CLI is capable of:
+- Importing your tests to Testomat.io
+- Add/update your tests IDs with your tests on the server
+- Removing the IDs if you don't need them
+
+This simple guide will show you how to synchronize your test codebase with Testomat.io.
+
+**Prerequisites:**  
+You need to add the **java-reporter** dependency of the actual version to your `pom.xml` file, as the CLI will add annotations from it to your  
+tests when the sync command runs:
+```xml
+<dependency>
+    <groupId>io.testomat</groupId>
+    <artifactId>java-reporter-junit</artifactId>
+    <version>0.7.4</version>
+</dependency>
+```
+or:
+```xml
+<dependency>
+    <groupId>io.testomat</groupId>
+    <artifactId>java-reporter-testng</artifactId>
+    <version>0.7.4</version>
+</dependency>
+```
+
+> **NOTE**: when you read this, the version might be updated (check it here: [JUnit](https://central.sonatype.com/artifact/io.testomat/java-reporter-junit) / [TestNG](https://central.sonatype.com/artifact/io.testomat/java-reporter-testng))
+---
+## Synchronizing your test codebase
+By default, this will import ***all*** the tests from the directory you run the query in and recursively from all the directories  
+inside. To change the directory, use the `--directory` property (see below).  
+There is a convenient way to do this in one move - use the oneliners that will download the latest version of  
+**java-check-tests** and run the `sync` command.  
+Here is one for UNIX/MAC users:
+```bash
+  export TESTOMATIO_URL=... && \
+  export TESTOMATIO=... && \
+  curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar && \
+  java -jar testomatio.jar sync
+```
+And this one is for Windows users:
+```cmd
+    set TESTOMATIO_URL=...&& ^
+    set TESTOMATIO=...&& ^
+    curl -L -O https://github.com/testomatio/java-check-tests/releases/latest/download/testomatio.jar&& ^
+    java -jar testomatio.jar sync
+```
+
+The `TESTOMATIO_URL` here is the server URL. In general, this will be `https://app.testomat.io` or `https://beta.testomat.io`.   
```

Comment:
```suggestion
The `TESTOMATIO_URL` here is the server URL. In general, this will be `https://app.testomat.io` or `https://beta.testomat.io`.   
```
Do not expose our beta!

TESTOMATIO_URL is needed only for on premise setups
By default it should be avoided